packages/wdio-utils/src/utils.ts

```diff
@@ -176,6 +178,21 @@ export async function safeImport (name: string): Promise<Services.ServicePlugin
          * imported correctly (for dev purposes).
          */
         const localNodeModules = path.join(process.cwd(), 'node_modules')
+
+        /**
+         * Determine the ESM import path by first using import.meta.url and then
+         * if not found by using the localNodeModules for the same reason as above.
+         */
+        try {
+            esmImportPath = await resolve(name, import.meta.url)
+        } catch (err: any) {
+            try {
+                esmImportPath = await resolve(name, pathToFileURL(localNodeModules).toString())
+            } catch (err: any) {
+                console.debug(`Could not determine esmImportPath using import.meta.url or localNodeModules for ${name}`)
+            }
+        }
```

Comment:
We don't want to do `await resolve(name, import.meta.url)` as we don't want users to load WebdriverIO dependencies, when they should load deps from their project.

Also we can remove `require.resolve` calls as `import-meta-resolve` also supports CJS modules. So I'ld suggest to replace `require.resolve` with `await resolve(...)`.

Reply:
Agreed